grafana-plugin/src/models/alert_receive_channel/alert_receive_channel.ts

```diff
@@ -415,26 +354,24 @@ export class AlertReceiveChannelStore extends BaseStore {
     });
   }
 
-  async getGrafanaAlertingContactPoints() {
-    return await makeRequest(`${this.path}contact_points/`, {}).catch(showApiError);
-  }
-
-  @action
-  async updateConnectedContactPoints(alertReceiveChannelId: AlertReceiveChannel['id']) {
-    const response = await makeRequest(`${this.path}${alertReceiveChannelId}/connected_contact_points `, {});
+  async fetchConnectedContactPoints(alertReceiveChannelId: ApiSchemas['AlertReceiveChannel']['id']) {
+    const { data } = await onCallApi().GET('/alert_receive_channels/{id}/connected_contact_points/', {
+      params: { path: { id: alertReceiveChannelId } },
+    });
 
     runInAction(() => {
       this.connectedContactPoints = {
         ...this.connectedContactPoints,
 
-        [alertReceiveChannelId]: response.reduce((list: ContactPoint[], payload) => {
-          payload.contact_points.forEach((contactPoint: { name: string; notification_connected: boolean }) => {
+        [alertReceiveChannelId]: data.reduce((list: ContactPoint[], payload) => {
+          // TODO: seems that backend sends contactPoint as string and on frontend we use it as { name: string, notification_connected: boolean }
```

Comment:
@vadimkerr we need to verify that together. For now I kept existing logic on the frontend but either we have wrong logic on the frontend or openapi schema for `ContactPoint` is incorrect

Reply:
fixed in https://github.com/grafana/oncall/pull/3851/commits/d3c4eb76d20f9d04741f339419d6f7013e6d8fff